_posts/2019-10-29-design-at-hackillinois.md

```diff
@@ -0,0 +1,40 @@
+
+Design is important and all around us. It influences our experiences, our decision making, our mood, and so many other things that you may not expect design to impact. With all these elements at play, design has the ability to transform the basic experience into the events that you talk about for months and years after it has happened. Designers steer the mindsets of those participating to get excited and be invested in open source, hackathons, and collaboration and keep them coming back for the following years. It’s more than posters hanging in the north quad, or the mobile app attendees use, or the amazing swag bags and clothing everyone gets excited about. It’s about creating an environment where the ease of great immersive design enhances students’ and mentors’ abilities to discover and generate incredible projects. Best said from Jan Hellemans, “We’re not designing products anymore, we’re designing systems that eventually will influence human expectations and enable a change in behavior.” Design at HackIllinois pushes the expectations of what open source can do for the world. 
+
+![A group of HackIllinois 2018 staff members take a picture in front of the photobooth in their staff t-shirts and jackets.](/assets/article_images/design-at-hackillinois/2018_photobooth.jpg)
+
+
+As an outsider in the coding community, it’s amazing to see the effect of design on attendees from all over the world. The curious build-up of branding and event designs is filled with thrill when attendees quickly look around to see the details in the Siebel and ECEB Building. When students ask if there are extra pins and stickers for their friends back home or to the intriguing UIUX design of web and mobiles, it definitely puts the cherry on top for all the hard work that our small but mighty team produces. 
+
+![The trophies given to the HackIllinois 2018 winners.](/assets/article_images/design-at-hackillinois/trophies.jpg)
+
+The next time you step into a hackathon, I urge you to take a few minutes and absorb all the design work put into it. What is the theme? What does it make you feel like? How do the tiniest of details enhance your time there? What gets you excited about that event? These answers may seem obvious at the time but take a step back and appreciate these aspects of events that may otherwise seem mandatory. Think about design and how important is it when it comes to amazing events like HackIllinois and how it transforms your experience. Good design is appealing, but exceptional design is what makes HackIllinois, HackIllinois.
```

Comment:
```suggestion
The next time you step into a hackathon, I urge you to take a few minutes and absorb all the design around you. What is the theme? How does it make you feel? What gets you excited about that event? How do the details, small or big, enhance your experience? While these answers may not seem obvious at the time, take a step back and you will learn to appreciate aspects of event planning that may otherwise seem ordinary. Think about design and how important it is in transforming your experience at events like HackIllinois. Good design is appealing, but exceptional design is what makes HackIllinois, HackIllinois.
```

Reply:
- _"What does it make you feel like?"_ to _"How does it make you feel?"_
- _"absorb all the design work put into it"_ to _"absorb all the design around you"_. -- You want them to appreciate your work because it's tangible. It's harder to see the work put into it because we didn't talk about the design process in specific.
- _"enhance your time there"_  to "enhance your experience" -- word choice
- _"These answers may seem obvious at the time but take a step back and appreciate these aspects of events that may otherwise seem mandatory."_ to "While these answers may not seem obvious at the time, take a step back and you will learn to appreciate aspects of event planning that may otherwise seem ordinary." -- I'm not entirely sure what you were going for here. I thought the sentence didn't make a lot of sense so I restructured it. Let me know if this better captures your thought. 
- Made some other minor changes.